frontend/src/components/WofPopper.vue

```diff
@@ -0,0 +1,25 @@
+<template>
+    <div class="popper"> <slot></slot> </div>
+</template>
+
+<script>
+export default{
+    name: "WofPopper"
+}
+</script>
+
+<style lang="less">
+@import './common.less';
+
+.popper{
```

Comment:
Don't use display table.
Change width units to 'vw' and height units to 'vh'

Reply:
I haven't launched this but it also looks like it would be positioned in the top left corner taking up all the space possible. Should probably be displayed in the center of the viewport.